Source/SwiftLintFramework/Rules/Idiomatic/RedundantObjcAttributeRule.swift

```diff
@@ -1,12 +1,11 @@
-import Foundation
 import SourceKittenFramework
 
 private let kindsImplyingObjc: Set<SwiftDeclarationAttributeKind> =
     [.ibaction, .iboutlet, .ibinspectable, .gkinspectable, .ibdesignable, .nsManaged]
 
 private let privateACL: Set<SwiftDeclarationAttributeKind> = [.private, .fileprivate]
```

Comment:
Due to your last change (`!acl.isPrivate`), this attribute is not needed any more.